packages/blade/src/components/Box/BaseBox/types/propsTypes.ts

```diff
@@ -187,13 +187,51 @@ type StyledPropsBlade = Partial<
   >
 >;
 
+type BoxCallbackProps = Omit<
+  Platform.Select<{
+    web: {
+      /**
+       * **Warning**
+       *
+       * Make sure to not use Box when you want to create a trigger that performs action on hover.
+       * You would probably want to render it as `button` using `styled.button` instead.
+       *
+       * Use this for hoverable containers in cases like custom menus.
```

Comment:
Don't know if this would work for custom menus because a lot of interactive things won't work like keyboard navigation, a11y (eg. <kbd>Tab</kbd>)

Reply:
Users will have to handle these things themselves (focus management, etc.)

Reply:
yes yes I meant it can be used as base element there since now we can put `onMouse` events there. Keyboard events will anyway go to `window` or some focusable item like `link` that triggered the hover menu.

Reply:
we shall rather create eslint rules for all the warnings in jsdoc. because i doubt people barely read JSDoc unless stuck or need more details. Even i won't read when I'm typing in flow